plugins/woocommerce/.distignore

```diff
@@ -36,3 +36,4 @@ renovate.json
 tsconfig.*
 webpack.config.js
 client/legacy/
+client/admin/
```

Comment:
leaving a note for later, to investigate if we need to negate the exclusion for feature flag